tools/depends/native/python3/Makefile

```diff
@@ -1,6 +1,6 @@
-include ../../Makefile.include PYTHON3-VERSION
+include ../../Makefile.include PYTHON3-VERSION ../../download-files.include
 PLATFORM=$(NATIVEPLATFORM)
-DEPS= ../../Makefile.include Makefile PYTHON3-VERSION 01-distutil-flags.patch
+DEPS= ../../Makefile.include ../../download-files.include Makefile PYTHON3-VERSION 01-distutil-flags.patch
```

Comment:
whats the reasoning for DEPS only being selectively updated in some but not others to include the new include file?

Reply:
the ones that have it updated, were done manually, the rest via script ;)

Reply:
I tried to add it everywhere, hope I got all of them